packages/create-amplify/src/get_project_root.test.ts

```diff
@@ -72,6 +74,39 @@ void describe('getProjectRoot', () => {
     assert.equal(projectRoot, path.resolve(userInput));
   });
 
+  void it('prints warning if creation of project root failed and path is absolute', async () => {
+    process.env.npm_config_yes = 'false';
+    const userInput = 'some/absolute/path';
+    mock.method(AmplifyPrompter, 'input', () => Promise.resolve(userInput));
+    const expectedError = new AmplifyUserError('ProjectDirectoryCreateError', {
+      message: `Failed to create project directory`,
+      resolution: `Ensure that ${userInput} is the correct path and you have write permissions to this location.`,
+    });
+    fsMkDirSyncMock.mock.mockImplementationOnce(() =>
+      Promise.reject(expectedError)
+    );
+    await assert.rejects(
+      () => getProjectRoot(),
+      (err: Error) => {
+        const sampleStderr = `some random stderr
+before the actual error message
+${util.inspect(err, { depth: null })}
+and some after the error message`;
+
+        const actual = AmplifyError.fromStderr(sampleStderr);
+        assert.deepStrictEqual(actual?.name, expectedError.name);
+        assert.deepStrictEqual(actual?.message, expectedError.message);
+        assert.deepStrictEqual(actual?.resolution, expectedError.resolution);
+        return true;
+      }
+    );
+    assert.equal(fsMkDirSyncMock.mock.callCount(), 1);
+    assert.equal(
+      fsMkDirSyncMock.mock.calls[0].arguments[0],
+      path.resolve(userInput)
+    );
```

Comment:
something like below
```suggestion
    await assert.throws(
      () => getProjectRoot(),
      new AmplifyUserError('ProjectDirectoryCreateError', {
        message: `Failed to create project directory`,
        resolution: `Ensure that ${userInput} is the correct path and you have write permissions to this location.`,
      });
    );
    assert.equal(fsMkDirSyncMock.mock.callCount(), 1);
    assert.equal(
      fsMkDirSyncMock.mock.calls[0].arguments[0],
      path.resolve(userInput)
    );
```

Reply:
@Amplifiyer 

I've made a few modifications.
at [d7226b6](https://github.com/aws-amplify/amplify-backend/pull/1071/commits/d7226b6a974c18e0362af236f848bc51c0e1bc70)

Reply:
I came up with idea.
```
const expectedError = new AmplifyUserError('ProjectDirectoryCreateError', {
      message: `Failed to create project directory`,
      resolution: `Ensure that ${
        userInput
      } is the correct path and you have write permissions to this location.`,
    });
    fsMkDirSyncMock.mock.mockImplementationOnce(() => {
      throw expectedError;
    });

    await assert.rejects(() => getProjectRoot(), {
      name: expectedError.name,
      message: expectedError.message,
    });
```